cephfs/file.go

```diff
@@ -273,6 +273,22 @@ func (f *File) Fchown(user uint32, group uint32) error {
 	return getError(ret)
 }
 
+// Flchown changes the file/directory last access and modification time.
+//
+// Implements:
+//	int ceph_lchown(struct ceph_mount_info *cmount, const char *path, int uid,
+//				  int gid);
+func (f *File) Flchown(path string, user uint32, group uint32) error {
```

Comment:
Since this call operates on a path, rather than an open file there's no need for a "F" prefix.  Additionally, this doesn't need an open file so it is better off named "Lchown" and be part of the the "permissions.go" file, rather than be a method on File and in file.go.

Reply:
Thanks for the comments. I have renamed and moved the function to the correct file and will be writing tests now.

Reply:
Now, for the test, I have to mock symlinks, is it like [this](https://github.com/ceph/go-ceph/blob/master/cephfs/path_test.go#L203) that you are talking about?.

Reply:
Yes, you'll want to use the `Symlink` function to create symlinks. You'll also want to use Statx to get the statuses of the items we modify. I am writing a more extensive comment separately below.